runtime/westend/src/lib.rs

```diff
@@ -1926,43 +1910,47 @@ mod remote_tests {
 	}
 }
 
-mod init_state_migration {
+mod clean_state_migration {
 	use super::Runtime;
-	use frame_support::traits::OnRuntimeUpgrade;
-	use pallet_state_trie_migration::{AutoLimits, MigrationLimits, MigrationProcess};
+	use frame_support::{pallet_prelude::*, storage_alias, traits::OnRuntimeUpgrade};
+	use pallet_state_trie_migration::MigrationLimits;
+
 	#[cfg(not(feature = "std"))]
 	use sp_std::prelude::*;
 
+	#[storage_alias]
+	type AutoLimits = StorageValue<StateTrieMigration, Option<MigrationLimits>, ValueQuery>;
+
+	// Actual type of value is `MigrationTask<T>`, putting a dummy
+	// one to avoid the trait constraint on T.
+	// Since we only use `kill` it is fine.
+	#[storage_alias]
+	type MigrationProcess = StorageValue<StateTrieMigration, u32, ValueQuery>;
```

Comment:
I think you can also just delete the pallet prefix instead of defining each alias but this also works obviously.

Reply:
Thanks for the tip, did not think about this approach, with 3 value only I will still keep this version.